packages/app/src/systems/Home/pages/Home/Home.tsx

```diff
@@ -1,10 +1,41 @@
+import { Flex } from "@fuel-ui/react";
+
+import { useHome } from "../../hooks/useHome";
+import { AssetsTitle } from "../components/AssetsTitle/AssetsTitle";
+import { HomeActions } from "../components/HomeActions/HomeActions";
+
+import { BalanceWidget } from "~/systems/Account/components/BalanceWidget/BalanceWidget";
+import { AssetList } from "~/systems/Asset";
 import { Layout } from "~/systems/Core";
 
 export function Home() {
+  const { state, context } = useHome();
+
+  const balances = [...(context.account?.balances || [])];
+
   return (
-    <Layout title="Home">
+    <Layout title="Home" isLoading={state.hasTag("loading")}>
       <Layout.TopBar />
-      <Layout.Content>Helo</Layout.Content>
+      <Layout.Content>
+        <Flex direction="column" css={{ height: "100%" }}>
+          {state.hasTag("loadingAccount") || !context.account ? (
```

Comment:
Here you can add this in an XState selector to avoid re-renders.

```jsx
const selectors = {
  isLoading: (state) => state.hasTag('loadingAccount') || !state.context.account
}

const useFoo() => {
  const [state, ,service] = useMachine(fooMachine)
  const isLoading = useSelector(service, selectors.isLoading)
  
  // and here return it as context
  return {
    context: {
      isLoading
    }
  }
}
```